### DISCUSSION TITLE
No background and invisible controls when theme is set to dark

### DISCUSSION THREAD

**Discussion by dawemeister** [lepoco/wpfui#492] [Q&A] [answered]
I tried to create a basic project using the getting started tutorial.

Here is my App.xaml
![image](https://user-images.githubusercontent.com/19820212/211085299-ad520d30-3720-4688-8cde-b38b596c0cb3.png)

And here is my MainWindow:
![image](https://user-images.githubusercontent.com/19820212/211085392-8e3b4d2f-3406-46ef-b7f4-f0438228e58f.png)

And the result is this:
![image](https://user-images.githubusercontent.com/19820212/211085544-dc0ebd80-a58d-4e01-aa16-d5a97923fb17.png)

I also tried other controls, but nothing rendered at all.
Nuget package is installed, and no errors shown.
What did I do wrong?

OS: Windows 11 
Framework: .NET 6

**Comment by pomianowski** [comment, marked as answer]
You have not changed the `MainWindow` background but you have selected the **Dark** theme in `App.xaml`.  
Most likely, you have a white background and a white icon, thus nothing is visible.

For example, you can do it this way:
```xml
<Window
    ...
    xmlns:ui="http://schemas.lepo.co/wpfui/2022/xaml"
    Background="{ui:ThemeResource ApplicationBackgroundBrush}">
</Window>
```

**Reply by GronbergInfo** [reply]
This was my problem also, according to the get started, you "only"(?) need to add the app.xaml stuff, iirc there is nothing that says you need to add "Background="{ui:ThemeResource ApplicationBackgroundBrush}"" to the window.

**Reply by Jay-o-Way** [reply]
> [!IMPORTANT]
> This answer doesn't work anymore
